docs/resources.rst

```diff
@@ -19,23 +19,28 @@ General Resources
 Integrated (Ethereum) Development Environments
 ==============================================
 
-    * `Brownie <https://eth-brownie.readthedocs.io/en/stable/>`_
-        Python-based development and testing framework for smart contracts targeting the Ethereum Virtual Machine.
-
-    * `Dapp <https://dapp.tools/>`_
-        Tool for building, testing and deploying smart contracts from the command-line.
+    * `Hardhat <https://hardhat.org/>`_
+        Ethereum development environment with local Ethereum network, debugging features and plugin ecosystem.
 
     * `Foundry <https://github.com/foundry-rs/foundry>`_
         Fast, portable and modular toolkit for Ethereum application development written in Rust.
 
-    * `Hardhat <https://hardhat.org/>`_
-        Ethereum development environment with local Ethereum network, debugging features and plugin ecosystem.
-
     * `Remix <https://remix.ethereum.org/>`_
         Browser-based IDE with integrated compiler and Solidity runtime environment without server-side components.
 
+    * `Dapp <https://dapp.tools/>`_
+        Tool for building, testing and deploying smart contracts from the command-line.
+
+    * `Brownie <https://eth-brownie.readthedocs.io/en/stable/>`_
```

Comment:
Brownie is no longer maintained either, an is replaced by Ape. https://eth-brownie.readthedocs.io/en/stable/

Also, why did you move the entries around - it seems like they were in alphabetical order previously?

Reply:
I'll double check if it indeed was the rationale but I think I moved them around to be in the order of most used based on our survey. Do you thing that's better or should I change it back to the alphabetical order?

Reply:
Unless we specifically agreed to change the ordering based on survey popularity, we should just keep the alphabetical order like we do almost everywhere else - after all, the alphabet is not gonna change, whereas framework popularity most certainly will. Also, strange that we don't have Foundry here, which I'm pretty sure is no. 2, behind Hardhat (maybe even no. 1 now).

Reply:
Ahh, you're right. Let's not make a change that would mean we'd have to update it every year according to the survey. I'll rearrange it back in the previous order and check why Foundry is not there.